Aire/View/NibClass/SceneView.swift

```diff
@@ -223,4 +263,31 @@ class SceneView: NibView {
 			}
 		}
 	}
+    
+    // camera position for adding labels
+    func makeUpdateCameraPos(towards: SCNVector3, pov: SCNNode) {
```

Comment:
does this mean this method makes an update on camera position change? rename to something more descriptive

Reply:
it makes an update to the nodes, using the camera position, can def rename to just update nodes